src/main/java/model/Lotto.java

```diff
@@ -0,0 +1,20 @@
+package model;
+
+import view.Validator;
+
+import static constant.LottoConstants.*;
+public class Lotto {
```

Comment:
```suggestion
import static constant.LottoConstants.*;

public class Lotto {
```

패키지와 클래스의 구분을 가시성을 위해서라도 띄워주면 좋을 것 같아요

Reply:
넵! 수정하겠습니다